src/goCover.ts

```diff
@@ -305,7 +305,7 @@ export function toggleCoverageCurrentPackage() {
 		return goTest(testConfig).then(success => {
 			if (!success) {
 				showTestOutput();
-				return [];
+				return applyCodeCoverageToAllEditors(tmpCoverPath, testConfig.dir);
```

Comment:
We can skip this line altogether as we are returning the same outside this `if` block.